pyfolio/tests/test_timeseries.py

```diff
@@ -311,12 +311,13 @@ class TestStats(TestCase):
         (simple_month_rets, utils.MONTHLY, 0.006885932704891129)
     ])
     def test_annual_ret(self, returns, period, expected):
-        self.assertEqual(
+        assert_almost_equal(
```

Comment:
This test was failing on my computer `AssertionError: 0.15500998835658053 != 0.15500998835658075` it was an insignificant enough difference that I didn't see it as a bigger issue.